packages/manager-components/src/components/action-banner/action-banner.component.tsx

```diff
@@ -36,6 +43,7 @@ export function ActionBanner({ message, cta, onClick }: ActionBannerProps) {
         </OsdsText>
         <OsdsButton
           className="sm:mt-0 mt-4 sm:ml-4 ml-0"
+          data-testid="actionBanner-button"
```

Comment:
we can get the button by text, don't we ?

Reply:
Employing data-testid greatly enhances the ease of development, particularly as it's tailor-made for this purpose! I've consistently preferred utilizing this attribute over locators by DOM hierarchy, particularly in conjunction with ODS. Consider the scenario where the DOM structure changes, or ODS introduces breaking changes; or even the simple scenario where the text of a button changes. In such cases, tests relying on DOM locators may encounter failures, highlighting the robustness of data-testid.